src/java/org/apache/cassandra/transport/messages/StartupMessage.java

```diff
@@ -118,8 +121,29 @@ else if (compression.equals("lz4"))
             clientState.setDriverVersion(options.get(DRIVER_VERSION));
         }
 
-        if (DatabaseDescriptor.getAuthenticator().requireAuthentication())
+        IAuthenticator authenticator = DatabaseDescriptor.getAuthenticator();
+        if (authenticator.requireAuthentication()) {
```

Comment:
@tolbertam could you put curly braces on new line as done everywhere else?

Reply:
ah good catch, not sure how I missed this;  will fix!

Reply:
looked around and corrected this and another place I was not following convention.

Was wondering if it's ever been discussed to enforce this through checkstyle?  I suspect maybe it has and was rejected because of possible chaos to reconcile it;  These rules found ~595 violations but a bunch of them come from the same set of files, so maybe manageable.

```diff
❯ git diff
diff --git a/.build/checkstyle.xml b/.build/checkstyle.xml
index 8b81f21281..9bd22dc1ac 100644
--- a/.build/checkstyle.xml
+++ b/.build/checkstyle.xml
@@ -179,6 +179,10 @@
       <message key="matchxpath.match"
                value="'Deprecated annotation must provide 'since' value."/>
     </module>
+    <module name="LeftCurly">
+        <property name="option" value="nl"/>
+        <property name="tokens" value="ANNOTATION_DEF,CLASS_DEF,CTOR_DEF,ENUM_CONSTANT_DEF,INTERFACE_DEF,LITERAL_CASE,LITERAL_CATCH,LITERAL_DEFAULT,LITERAL_DO,LITERAL_ELSE,LITERAL_FINALLY,LITERAL_FOR,LITERAL_IF,LITERAL_SWITCH,LITERAL_SYNCHRONIZED,LITERAL_TRY,LITERAL_WHILE,METHOD_DEF,OBJBLOCK,STATIC_INIT,RECORD_DEF,COMPACT_CTOR_DEF"/>
+    </module>
   </module>

 </module>
```

Reply:
https://gist.github.com/tolbertam/2b0f6245c3e94a7809f716d99a4ec9c3 if interested in seeing the violations.  I removed `LAMBDA` as that one was violated quite a bit and somewhat reasonable to allow curly braces on same line there for oneliners.

Reply:
caught a couple other places I missed..should be good now that I've run checkstyle against my changeset.